include/baresip.h

```diff
@@ -151,6 +151,22 @@ struct call  *call_find_linenum(const struct list *calls, uint32_t linenum);
 void call_set_current(struct list *calls, struct call *call);
 
 
+/*
+* Custom headers
+*/
+const struct list *call_get_custom_hdrs(const struct call *call);
+typedef int (custom_hdrs_h)(const struct pl *name, const struct pl *val,
+	void *arg);     //returns error code if any
+int custom_hdrs_alloc(struct list **hdrs);
```

Comment:
please insert some empty lines here, between the typedef and
the function prototypes.

also please use C89 style comments (`/* xxx */`)